File: backend/SprintBusiness/SprintBuisness.Contracts/Whatsapp/Dtos/MessageResponse/SignalRFlowMessageListItemResponse.cs
using SprintBusiness.Domain.Flows.FlowMessageLists.Keys;
using SprintBusiness.Domain.Flows.FlowMessages.Keys;

namespace SprintBuisness.Contracts.Whatsapp.Dtos.MessageResponse
{
    public class SignalRFlowMessageListItemResponse
    {
        public required FlowMessageListItemId Id { get; set; }
        public required string Content { get; set; }
        public string? Description { get; set; }
        public required FlowMessageId Next { get; set; }
    }
}

File: frontend/src/app/core/helpers/signalr/SignalRFlowMessageListItemResponse.ts
import FlowMessageListItemId from "../../entities/flow-messages/flow-message-list/keys/FlowMessageListItemId";
import FlowMessageId from "../../entities/flow-messages/flow-message/keys/FlowMessageId";

export interface SignalRFlowMessageListItemResponse {
  id: FlowMessageListItemId;
  content: string;
  description?: string;
  next: FlowMessageId;
}
